web/ui/src/app/lineage/graph/graph.model.ts

```diff
@@ -47,16 +47,24 @@ export abstract class VisNode implements vis.Node {
                 public icon: any,
                 public type: VisNodeType) {
     }
+    
```

Comment:
Another redundant method. Please take a look at LoDash `clone()` function. There are four variants of it.

Reply:
corrected